index.js

```diff
@@ -135,31 +139,31 @@ class Plugin extends PluginMiniAccounts {
   }
 
   _handleIncomingPrepare (account, ilpData) {
-    const { amount } = IlpPacket.deserializeIlpPrepare(ilpData)
+    const amount = new EthUnit(
+      IlpPacket.deserializeIlpPrepare(ilpData).amount,
+      'gwei'
+    )
 
     const secured = account.getSecuredBalance()
     const prepared = account.getBalance()
     const newPrepared = prepared.add(amount)
     const unsecured = newPrepared.sub(secured)
-    debug(unsecured.toString(), 'unsecured; secured balance is',
-      secured.toString(), 'prepared amount', amount, 'newPrepared',
-      newPrepared.toString(), 'prepared', prepared.toString())
 
-    if (unsecured.greaterThan(this._bandwidth)) {
```

Comment:
I don't think the connector can set this, because it doesn't track the unsecured per account in the multi-plugin